src/components/FileVideo.tsx

```diff
@@ -56,46 +68,56 @@ const FileVideo: FC<FileVideoProps> = ({ filePath, itemId, mimetype }) => {
   };
 
   useEffect(() => {
-    navigation.setOptions({
-      headerRight: () => (
-        <View>
-          {isDownloading ? (
-            <View style={styles.headerButtonsDownloadingState}>
-              <Button
-                disabled
-                disabledStyle={{ backgroundColor: PRIMARY_COLOR }}
-                buttonStyle={{ backgroundColor: PRIMARY_COLOR }}
-                icon={
-                  <MaterialIcons
-                    name={'cloud-download'}
-                    color="#ffffff"
-                    size={25}
-                  />
-                }
-              ></Button>
-            </View>
-          ) : (
-            <View style={styles.headerButtons}>
-              <Button
-                buttonStyle={{ backgroundColor: PRIMARY_COLOR }}
-                icon={
-                  <MaterialIcons name={'save-alt'} color="#ffffff" size={25} />
-                }
-                onPress={handleSaveFileFromS3Url}
-              ></Button>
-              <Button
-                buttonStyle={{ backgroundColor: PRIMARY_COLOR }}
-                icon={
-                  <MaterialIcons name={'ios-share'} color="#ffffff" size={25} />
-                }
-                onPress={handleShareFileFromS3Url}
-              ></Button>
-            </View>
-          )}
-        </View>
-      ),
-    });
-  }, [isDownloading]);
+    if (!isPlayerView) {
+      navigation.setOptions({
+        headerRight: () => (
+          <View>
+            {isDownloading ? (
+              <View style={styles.headerButtonsDownloadingState}>
+                <Button
+                  disabled
```

Comment:
Not really fan of this logic in this component. Would that make sense to have a "Wrapper" component that you can reuse? I feel like you mix the "File-" components being reusable components & screens. I think it would be easier if you had a screen component that would obviously not be included when we're in Player.